package.json

```diff
@@ -15,7 +15,7 @@
   "main": "index.js",
   "scripts": {
     "lint": "eslint .",
-    "test": "mocha --timeout 30000 test/*test.js",
```

Comment:
did we move these flag/config options to `mocha.opts` file or do you feel it's not needed because we're using local fixtures for the requests instead of expensive external requests?

Reply:
The timeout flag is no longer needed because the local stub server is significantly faster than the real Google Maps API.

The second argument `test/*test.js` was never needed, but if you like, I can bring it back.

Reply:
I just wanted to understand the reasoning (my comment was mainly for the timeout), I understand we don't need to specify `test/*test.js` 👍